examples/rails/README.md

```diff
@@ -0,0 +1,262 @@
+    ```
+    $ ./bin/rails db:migrate
+    == 20210803025742 CreateArticles: migrating ===================================
+    -- create_table(:articles)
+      -> 23.5728s
+    == 20210803025742 CreateArticles: migrated (23.5729s) =========================
+    ```
+
+### Use the CLI to interact with the database
+1. Run the following command to start `irb`:
+    ```shell
+    bin/rails console
+    ```
+1. At the prompt, initialize a new `Article` object:
+    ```ruby
+    article = Article.new(title: "Hello Rails", body: "I am on Rails!")
+    ```
+1. Run the following command to save the object to the database:
+    ```ruby
+    article.save
+    ```
+1. Review the object and you can see the field `id`, `created_at`, and `updated_at` have been set:
+    ```ruby
+    article
+    ```
+    Sample output:
+    ```
+    => #<Article id: 4170057092403543076, title: "Hello Rails", body: "I am on Rails!", created_at: "2021-08-03 03:06:26.096275000 +0000", updated_at: "2021-08-03 03:06:26.096275000 +0000">
+    ```
+1. You can find `Article.find(id)` or `Article.all` to fetch data from the database. For example:
+    ```ruby
+    irb(main):007:0> Article.find(4170057092403543076)
+    Article Load (49.2ms)  SELECT `articles`.* FROM `articles` WHERE `articles`.`id` = @p1 LIMIT @p2
+    => #<Article id: 4170057092403543076, title: "Hello Rails", body: "I am on Rails!", created_at: "2021-08-03 03:06:26.096275000 +0000", updated_at: "2021-08-03 03:06:26.096275000 +0000">
+
+    irb(main):008:0> Article.all
+    Article Load (73.9ms)  SELECT `articles`.* FROM `articles` /* loading for inspect */ LIMIT @p1
+    => #<ActiveRecord::Relation [#<Article id: 4170057092403543076, title: "Hello Rails", body: "I am on Rails!", created_at: "2021-08-03 03:06:26.096275000 +0000", updated_at: "2021-08-03 03:06:26.096275000 +0000">]>
+    ```
+
+### Update the app to show a list of records
+1. Use the controller generator to create a controller:
+    ```shell
+    bin/rails generate controller Articles index
+    ```
+1. Open the file `app/controllers/articles_controller.rb`, and change the `index` action to fetch all articles from the database:
+    ```ruby
+    class ArticlesController < ApplicationController
+      def index
+        @articles = Article.all
+      end
+    end
+    ```
+1. Open `app/views/articles/index.html.erb`, and update the file as the following:
+    ```html
+    <h1>Articles</h1>
+    <ul>
+      <% @articles.each do |article| %>
+        <li>
+          <%= article.title %>
+        </li>
+      <% end %>
+    </ul>
+    ```
+1. Run your server again
+    ```shell
+    ./bin/rails s
+    ```
+1. In your browser, naviate to the URL [http://localhost:3000/articles/index](http://localhost:3000/articles/index). And you will see the `Hello Rails` record you entered previously.
+1. [Optional] you can follow the rest of the steps in the [Getting Started with Rails](https://guides.rubyonrails.org/getting_started.html) guide.
+
+### Cleaning up
```

Comment:
Nits: `Clean up`? Be consistent with other titles.

Reply:
Updated.